src/TwigFactory.php

```diff
@@ -51,10 +51,10 @@ public function create( array $loaders ): Twig_Environment {
 	}
 
 	public function newWikiPageLoader( WikiContentProvider $provider ) {
-		if ( empty( $this->config['loaders']['wiki'] ) ) {
+		if ( !$this->config['loaders']['wiki']['enabled'] ) {
 			return null;
 		}
-		return new TwigPageLoader( $provider );
+		return new TwigPageLoader( $provider, $this->config['loaders']['wiki']['rawpages'] ?? [] );
```

Comment:
If you omit the field, the recursive replace will not replace it, and it will end up being the default.